app/models/vacancy.rb

```diff
@@ -253,16 +249,11 @@ def skip_update_callbacks?
   end
 
   def show_supporting_documents?
-    UploadDocumentsFeature.enabled? && !show_candidate_specification?
-  end
-
-  def show_candidate_specification?
-    # TODO: This method should change when the behaviour for changing candidate specification is defined
-    experience.present? && qualifications.present? && education.present?
+    UploadDocumentsFeature.enabled? && documents.any?
   end
 
-  def show_key_info_in_search_snippet?
```

Comment:
We need this method to remain in place for this release.  I'll leave it to you @cesidio as to how to do it.